lib/services/rpcbind.pm

```diff
@@ -39,13 +39,15 @@ sub config_service {
 }
 
 sub enable_service {
-    common_service_action('rpcbind',   $service_type, 'enable');
-    common_service_action($nfs_server, $service_type, 'enable');
+    common_service_action('rpcbind',        $service_type, 'enable');
```

Comment:
Only the socket unit should get enabled and started.
_____________________________________________-
From dev's description, I think we only need start and enable rpcbind.socket, WDYT?

Reply:
changed and verified  https://openqa.nue.suse.com/tests/6353095